sync-core/src/main/java/com/cloudant/sync/replication/ReplicatorBuilder.java

```diff
@@ -109,6 +120,44 @@ public E from(S source) {
     }
 
     /**
+     * Variable argument version of {@link #addRequestFilters(List)}
+     * @param filters The request filters to add.
+     * @return The current instance of {@link ReplicatorBuilder}
+     */
+    public E addRequestFilters(HttpConnectionRequestFilter ... filters){
+        return addRequestFilters(Arrays.asList(filters));
+    }
+
+    /**
+     * Adds filters to the list of request filters to use for each request made by this replication.
+     * @param filters The request filters to add.
+     * @return The current instance of {@link ReplicatorBuilder}
+     */
+    public E addRequestFilters(List<HttpConnectionRequestFilter> filters){
+        this.requestFilters.addAll(filters);
+        return (E)this;
```

Comment:
Intellij issues warnings for these casts, which I don't think we can avoid. Maybe add statement level ignores (`//noinspection unchecked`)?